### PR TITLE
rootfs-builder: exclude unsupported archs from euleros and clearlinux

### PR DESCRIPTION
**Pull request by Pennyzct**
# Description of problem
For clearlinux, for now, it is only designed for amd64.
For euleros, it has supported aarch64 starting from 2.3, but here is the sad part. 
there existed bugs in their 2.3.x image, this bug existed in both x86_64 and aarch64 image. related issue [euleros/euleros-docker-images/#13](https://github.com/euleros/euleros-docker-images/issues/13) has been raised.
since x86_64 is using version 2.2, it's fine there, stay on that.😎

### CODE REVIEW

File: rootfs-builder/clearlinux/config.sh [hunk 1]
```diff
@@ -26,6 +26,6 @@ PACKAGES="util-linux-bin iptables-bin libudev0-shim chrony"
 INIT_PROCESS=systemd
 # List of zero or more architectures to exclude from build,
 # as reported by  `uname -m`
-ARCH_EXCLUDE_LIST=(ppc64le)
+ARCH_EXCLUDE_LIST=( aarch64 ppc64le )
```
Comment (jodh-intel) [inline rootfs-builder/clearlinux/config.sh]: For completeness, I wonder if it's also worth adding `s390x`?

Comment (Pennyzct) [reply rootfs-builder/clearlinux/config.sh]: yep, you are right. ;)
and also for completeness, the latest euleros v2.3 only supports amd64 and arm64. you could check it from their official [docker hub repo](https://hub.docker.com/_/euleros)

**Pull request merged**